backend/app/services/splunk.py

```diff
@@ -110,6 +109,60 @@ async def _stream_results(self, oneshotsearch_results):
         for record in results.JSONResultsReader(oneshotsearch_results):
             yield record
 
+    async def filterPost(self, query, searchList=""):
+        """
+        Query data to construct filter from splunk server using splunk lib sdk
+
+        Args:
+            query (string): splunk query
+            OPTIONAL: searchList (string): additional query parameters for index
+        """
+
+        try:
+            # If additional search parameters are provided, include those in searchindex
+            searchindex = (
+                "search index={} {}".format(self.indice, searchList)
+                if searchList
+                else "search index={}".format(self.indice)
+            )
+            # cluster_artifacts.cpu_models{}.cpu
+            search_query = (
+                "search index={} {} | stats count AS total_records, values(cpu) AS cpu, values(node_name) AS nodeName, values(test_type) AS benchmark, values(ocp_version) AS ocpVersion, values(ocp_build) AS releaseStream".format(
+                    self.indice, searchList
+                )
+                if searchList
+                else "search index={} | stats count AS total_records, values(cpu) AS cpu, values(node_name) AS nodeName, values(test_type) AS benchmark, values(ocp_version) AS ocpVersion, values(ocp_build) AS releaseStream".format(
+                    self.indice
+                )
+            )
```

Comment:
This kind of ternary expression turns out kinda ugly when wrapped. It might be easier to read as an `if`/`else`.

Reply:
Sure!